python/foglamp/plugins/south/envirophat/envirophat.py

```diff
@@ -33,6 +32,52 @@
         'type': 'integer',
         'default': '1000'
     },
+    'assetPrefix': {
```

Comment:
These names differ from those in the Jira - the names should really match as the idea is to have a comment set of names across all the plugins.

Reply:
Actually, full asset name is composed of a prefix (e.g. envirophat) + sensor name (e.g. weather). Full name (e.g. envirophat_temperature) is called asset_code in readings table. Hence it is mentioned as assetPrefix and not assetName. This is the convention we have followed in sensortag and other plugins. However, to match Jira, it will be renamed to assetName instead of assetPrefix. But it will better to follow an uniform convention in all plugins, whether assetName or assetPrefix or assetCode or whatever.